assisted-events-scrape/events_scrape/logger.py

```diff
@@ -3,57 +3,30 @@
 import os
 import re
 import sys
-
-
-class SensitiveFormatter(logging.Formatter):
```

Comment:
this thing no longer needed, right?

Reply:
AFAIU this functionality has never been used in this project. Maybe it's a copy paste from test-infra, where it might be necessary to protect secrets?

Reply:
yes, it's possible